android/modules/ui/res/values/attrs.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<attr name="headerTitleSize" format="string" />
+	<attr name="headerTitleStyle" format="string" />
+	<attr name="headerTitleColor" format="color" />
+	<attr name="headerTitleBackground" format="reference" />
+	<attr name="headerTitleBackgroundColor" format="color" />
+	<attr name="footerTitleSize" format="string" />
+	<attr name="footerTitleStyle" format="string" />
+	<attr name="footerTitleColor" format="color" />
+	<attr name="footerTitleBackground" format="reference" />
+	<attr name="footerTitleBackgroundColor" format="color" />
+</resources>
```

Comment:
The new attributes you are adding should be prefixed to avoid name collision.
Perhaps with "ti"? Ex: `tiHeaderTitleSize`

I don't see a consistent naming convention for attributes from Google. Sometimes they camel case it and in other places they snake case it with underscores.